contracts/upgradeable_contracts/multi_amb_erc20_to_erc677/HomeMultiAMBErc20ToErc677.sol

```diff
@@ -183,17 +212,19 @@ contract HomeMultiAMBErc20ToErc677 is
         // When transferFrom is called, after the transfer, the ERC677 token will call onTokenTransfer from this contract
         // which will call passMessage.
         require(!lock());
-        address to = address(this);
         // if msg.sender if not a valid token contract, this check will fail, since limits are zeros
         // so the following check is not needed
         // require(isTokenRegistered(token));
         require(withinLimit(token, _value));
         addTotalSpentPerDay(token, getCurrentDay(), _value);
 
+        uint256 balanceBefore = token.balanceOf(address(this));
         setLock(true);
-        token.transferFrom(msg.sender, to, _value);
+        token.transferFrom(msg.sender, address(this), _value);
         setLock(false);
-        bridgeSpecificActionsOnTokenTransfer(token, msg.sender, _receiver, _value);
+        uint256 balanceDiff = token.balanceOf(address(this)).sub(balanceBefore);
```

Comment:
do we need these actions for the the Home contracts? We are working with our version of ERC677 token and we know for sure that the balance will be changed by the same value which is specified in the `transferFrom` method call, don't we?

Reply:
Agree, seems like for half-duplex OB there is not need in such changes on Home side.